### PR TITLE
WIP: Webapp Refactoring

### PR DESCRIPTION
**Pull request by chetanyakan**

### CODE REVIEW

File: webapp/babel.config.js [hunk 1]
```diff
@@ -1,6 +1,3 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
```
Comment (manojmalik20) [inline webapp/babel.config.js:3]: Why remove these comments?

File: webapp/src/components/wopi_file_preview.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        const {fileInfo} = props;
+        if (fileInfo?.id) {
+            handleWopiFile(fileInfo.id);
+        }
+    }, [props.fileInfo?.id]);
+
+    const handleWopiFile = async (fileID: string) => {
+        //ask the server for the Collabora Online URL & token where the file will be edited
+        //and load it into the iframe
+        const fileData = await Client.getCollaboraOnlineURL(fileID);
```
Comment (hrishavkmr) [inline webapp/src/components/wopi_file_preview.tsx:22]: what if the API call fails?

**Review by hrishavkmr — APPROVED** [review]

File: webapp/babel.config.js (hunk 1 above)
Comment (chetanyakan) [reply webapp/babel.config.js:3]: Not needed. Were added while syncing with starter template.

File: webapp/src/components/wopi_file_preview.tsx (hunk 2 above)
Comment (chetanyakan) [reply webapp/src/components/wopi_file_preview.tsx:23]: Added a TODO to handle that later.

**Pull request merged**